translator/jsonconfig/mergeJsonConfig.go

```diff
@@ -16,20 +17,29 @@ import (
 	"github.com/aws/amazon-cloudwatch-agent/translator/util"
 )
 
-func MergeJsonConfigMaps(jsonConfigMapMap map[string]map[string]interface{}, defaultJsonConfigMap map[string]interface{}, multiConfig string) (map[string]interface{}, error) {
+func MergeJsonConfigMaps(jsonConfigMapMap map[string]map[string]interface{}, defaultJsonConfigMap map[string]interface{}, multiConfig string, strictValidation bool, inputJsonDirPathErr error ) (map[string]interface{}, error) {
 	if jsonConfigMapMap == nil || len(jsonConfigMapMap) == 0 {
 		if os.Getenv(config.USE_DEFAULT_CONFIG) == config.USE_DEFAULT_CONFIG_TRUE {
```

Comment:
Should we honor this logic for a non ECS environment? I don't know the context behind this, but it seems like it could be relevant.

Reply:
Either way it's the same meaning. Therefore, it is possible to honor this logic for a non ECS and ECS environment. Besides, I only want to specific about log messages as possible.